docs/generated/packages/cypress/executors/cypress.json

```diff
@@ -24,12 +24,14 @@
         "type": "string",
         "description": "The path of the Cypress configuration json file.",
         "x-completion-type": "file",
-        "x-completion-glob": "cypress?(*)@(.js|.ts|.json)"
+        "x-completion-glob": "cypress?(*)@(.js|.ts|.json)",
+        "x-priority": "important"
       },
       "watch": {
         "type": "boolean",
         "description": "Recompile and run tests when files change.",
-        "default": false
+        "default": false,
+        "x-priority": "important"
```

Comment:
This is kind of mean because you were so fast with this PR. But we decided that `watch` should not be marked important across executors for now. 
You can have a look at the 'Establish guidelines for x-priority' task in asana for reference